spec/integration/action_spec.rb

```diff
@@ -1,31 +1,150 @@
 require 'verbalize/action'
 
 describe Verbalize::Action do
-  describe '.verbalize' do
+  let(:a) { double(divide_by: 20) }
+
+  let(:simple_action) do
+    Class.new do
+      include Verbalize::Action
+
+      input :a, :b, :should_fail
+
+      def call
+        fail! 'Are you crazy?!? You can’t divide by zero!' if should_fail
+        a.divide_by(b)
+      end
+    end
+  end
+
+  describe '.call' do
     it 'returns a Verbalize::Failure instance on fail!' do
-      some_class = Class.new do
+      result = simple_action.call(a: a, b: 10, should_fail: true)
+
+      expect(result).to be_an_instance_of(Verbalize::Failure)
+      expect(result).to be_failure
+      expect(result.failure).to eq 'Are you crazy?!? You can’t divide by zero!'
+    end
+
+    it 'returns a Verbalize::Success instance on success' do
+      result = simple_action.call(a: a, b: 10, should_fail: false)
+
+      expect(result).to be_an_instance_of(Verbalize::Success)
+      expect(result).to be_success
+      expect(result.value).to eq 20
+    end
+
+    it 'short circuits the call block on failure' do
+      simple_action.call(a: a, b: 10, should_fail: true)
+
+      expect(a).not_to have_received(:divide_by)
+    end
+
+    it 'stubbed failures return a Verbalize::Falure on `call`' do
```

Comment:
I don't know what dictionary you are using, that is definitely correct. :trollface: